x-pack/platform/plugins/shared/onechat/public/flyout/open_conversation_flyout.tsx

```diff
@@ -78,6 +80,10 @@ export function openConversationFlyout(
       type: 'push',
       hideCloseButton: true,
       'aria-labelledby': ariaLabelledBy,
+      maxWidth: 1200, // Maximum width for resizable flyout to prevent NaN error
+      css: css`
+        z-index: ${euiThemeVars.euiZFlyout + 3};
+      `,
```

Comment:
NIT: inconsistency, the two others are `${euiThemeVars.euiZFlyout + 4}` and this one is `+3` (just for grep's sake)